deployments/hadoop-yarn/bin/run-tests.sh

```diff
@@ -53,20 +53,44 @@
 
 # -----------------------------------------------------
 # Run Benchmarks
+ 
+    # Create Users
+
+    source ${treetop:?}/zeppelin/bin/create-user-tools.sh
+    count=1
+    testers=()
+    while [ "$count" -le $num_users ]; do
+        testers[$count]=$(pwgen 8 1)
+        let "count += 1"
+    done
+
+    tempfile=$(mktemp)
+
+ testusername=$(
+        pwgen 8 1
+        )
+
+    createusermain \
+        "${testusername}" \
+    | tee "/tmp/${testusername}.json" | jq '.'
```

Comment:
Is this left over from a previous experiment? It creates a single user, which doesn't seem to be used later on.